### ISSUE TITLE
fix: No named parameter with the name 'offLabelColor' from CupertinoSwitch

### ISSUE THREAD

**Issue by emanuel-braz** [shorebirdtech/shorebird#1632]
**Description**
I have configured shorebird in my project, but it is not compiling with the command `shorebird release android -- --flavor=development -t lib/main_dev.dart`

I noticed that shorebird `0.21.1` uses `Flutter 3.16.4-0.0.pre.59` cached from source code(I expected it to be 3.16.5).
I have already tried using `Flutter 3.16.4` and `Flutter 3.16.5` as well, but I was unsuccessful.

The project depends on the `flutter_platform_widgets` package that uses `CupertinoSwitch.offLabelColor`, which is not found.

If I use regular `flutter build apk...` it builds the app without errors.

The error
```
../../../../.puro/shared/pub_cache/hosted/pub.dev/flutter_platform_widgets-6.0.2/lib/src/platform_switch.dart:197:7: Error: No named parameter with the name 'offLabelColor'.
      offLabelColor: data?.offLabelColor,
      ^^^^^^^^^^^^^
../../../../.shorebird/bin/cache/flutter/4e8a7c746ae6f10951f3e676f10b82b21d7300a5/packages/flutter/lib/src/cupertino/switch.dart:68:9: Context: Found this candidate, but the arguments don't match.
  const CupertinoSwitch({
        ^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script '/Users/eb/.shorebird/bin/cache/flutter/4e8a7c746ae6f10951f3e676f10b82b21d7300a5/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 1373

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDevelopmentRelease'.
> Process 'command '/Users/eb/.shorebird/bin/cache/flutter/4e8a7c746ae6f10951f3e676f10b82b21d7300a5/bin/flutter'' finished with non-zero exit value 1
```

<details>
<summary>shorebird doctor -v</summary>
Shorebird 0.21.1 • git@github.com:shorebirdtech/shorebird.git
Flutter 3.16.5 • revision 4e8a7c746ae6f10951f3e676f10b82b21d7300a5
Engine • revision 001b60083b642c134c62e7c10bc34f188b0a2869

Android Toolchain
  • Android Studio: /Applications/Android Studio.app/Contents
  • Android SDK: /Users/eb/Library/Android/sdk
  • ADB: /Users/eb/Library/Android/sdk/platform-tools/adb
  • JAVA_HOME: /Applications/Android Studio.app/Contents/jbr/Contents/Home

✓ Shorebird is up-to-date (0.4s)
✓ Flutter install is correct (0.6s)
✓ AndroidManifest.xml files contain INTERNET permission (31ms)
✓ Has access to storage.googleapis.com (0.1s)
</details>

<details>
<summary>flutter doctor -v</summary>
[✓] Flutter (Channel stable, 3.16.4, on macOS 13.0 22A8380 darwin-arm64, locale en-BR)
    • Flutter version 3.16.4 on channel stable at /Users/eb/.puro/envs/3.16.4/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2e9cb0aa71 (5 weeks ago), 2023-12-11 14:35:13 -0700
    • Engine revision 54a7145303
    • Dart version 3.2.3
    • DevTools version 2.28.4

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
    • Android SDK at /Users/eb/Library/Android/sdk
    • Platform android-34, build-tools 33.0.2
    • ANDROID_HOME = /Users/eb/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14C18
    • CocoaPods version 1.12.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] VS Code (version 1.85.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.81.20231227

[✓] Connected device (3 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554 • android-arm64  • Android 13 (API 33) (emulator)
    • macOS (desktop)             • macos         • darwin-arm64   • macOS 13.0 22A8380 darwin-arm64
    • Chrome (web)                • chrome        • web-javascript • Google Chrome 120.0.6099.216

[✓] Network resources
    • All expected network resources are available.
</details>

**Comment by eseidel** [comment]
Correct, we made an error in merging Flutter into our fork in our last release.  This will be fixed in our next release (which we're working on for today, but might slip to tomorrow).

**Comment by eseidel** [comment]
We didn't quite get this released today, but should go out tomorrow morning.  Thanks for your patience.

**Comment by eseidel** [comment]
This release has been held up by one more (unrelated) crasher, we're hoping to get it out tomorrow. 🤞

**Comment by eseidel** [comment]
We believe this to be resolved in 0.22.0.
